draft-ietf-taps-interface.md

```diff
@@ -1356,9 +1423,11 @@ functionality to implement Clone. In that case, entangled Connections are
 multiplexed together, giving them similar treatment not only inside endpoints
 but also across the end-to-end Internet path.
 
-Note that calling Clone() may result in on-the-wire signaling, e.g., to open a new connection, depending on the underlying Protocol Stack.
+Note that calling Clone() may result in on-the-wire signaling, e.g., to open a new connection, depending on the underlying Protocol Stack. In this case, the transport system will ensure consistency of
+Connection Properties by uniformly applying them to all underlying connections
+in a group.
```

Comment:
It has to do with "e.g., to open a new connection"; in this case refers to opening new connections instead of using multiple streams (e.g., when multi-streaming is not available).

Reply:
Oh, that makes sense. Could you change "In this case" to "When opening new connections" then, please?